### PR TITLE
WALMA-15: Adding user, security and workflow related shortcuts

### PR DESCRIPTION
**Pull request by dministro**
WALMA-15

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ jobs:
     with:
       machine-types: "[\"ubuntu-latest\", \"windows-latest\"]"
       timeout-minutes: 60
+      verify-dotnet-consolidation: false
```
Comment (Piedone) [inline .github/workflows/build-and-test.yml]: I assume this is temporary and you'll remove it before this is ready for review?

**Comment by Piedone** [comment]
Use a PR title that makes sense in the context of OSOCE.

File: .github/workflows/build-and-test.yml (hunk 1 above)
Comment (dministro) [reply .github/workflows/build-and-test.yml]: Yes, this is temporary

**Comment by Luko6** [comment]
Fix conflicts

**Pull request merged**